mantle/cmd/kola/testiso.go

```diff
@@ -66,6 +66,11 @@ var (
 	isOffline        bool
 	isISOFromRAM     bool
 
+	// These tests only run on RHCOS
```

Comment:
We should try to push forward on https://github.com/coreos/fedora-coreos-tracker/issues/302. Even if it's not "true FIPS", it'd be valuable for testing purposes like here. Based on [the latest comment there](https://github.com/coreos/fedora-coreos-tracker/issues/302#issuecomment-1460506911), things Should Work? (It seems like the main thing is upstreaming the RHCOS FIPS glue to FCOS.)

Reply:
(To be clear: not a blocker, just an aside.)

Reply:
Yes.

(Incidentally we should try to semi-standardize the terminology for "this is a non-blocking comment"...)